tracer/src/Datadog.Trace/Ci/Agent/MessagePack/SpanEventMessagePackFormatter.cs

```diff
@@ -0,0 +1,36 @@
+// <copyright file="SpanEventMessagePackFormatter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Ci.EventModel;
+using Datadog.Trace.Vendors.MessagePack;
+
+namespace Datadog.Trace.Ci.Agent.MessagePack
+{
+    internal class SpanEventMessagePackFormatter : EventMessagePackFormatter<SpanEvent>
+    {
+        public override int Serialize(ref byte[] bytes, int offset, SpanEvent value, IFormatterResolver formatterResolver)
+        {
+            if (value is null)
+            {
+                return 0;
+            }
+
+            var originalOffset = offset;
+
+            offset += MessagePackBinary.WriteMapHeader(ref bytes, offset, 3);
+
+            offset += MessagePackBinary.WriteStringBytes(ref bytes, offset, TypeBytes);
+            offset += MessagePackBinary.WriteString(ref bytes, offset, value.Type);
+
+            offset += MessagePackBinary.WriteStringBytes(ref bytes, offset, VersionBytes);
+            offset += MessagePackBinary.WriteInt32(ref bytes, offset, 1);
```

Comment:
Can we define the version value in the event type itself instead of hard-coding here?

Reply:
done in [af479e5](https://github.com/DataDog/dd-trace-dotnet/pull/2402/commits/af479e5ffb8977907a65bc844d114411cddf606c) for CIVisibilityEvent ojects (Spans and Tests)